src/lib/Containers/Inbox.tsx

```diff
@@ -47,12 +47,16 @@ export class Inbox extends React.Component<Props, State> {
 
     this.setState({ fetchingData: true })
 
-    // Allow Conversations & Active Bids to properly force-fetch themselves.
-    // The stored refs are the Relay containers; the components themselves are nested under as refs.
-    this.activeBids.refs.component.refreshActiveBids()
-    this.conversations.refs.component.refreshConversations(() => {
-      this.setState({ fetchingData: false })
-    })
+    if (this.activeBids && this.conversations) {
+      // Allow Conversations & Active Bids to properly force-fetch themselves.
+      // The stored refs are the Relay containers; the components themselves are nested under as refs.
+      this.activeBids.refs.component.refreshActiveBids()
+      this.conversations.refs.component.refreshConversations(() => {
```

Comment:
Can you add an error handler here in a similar vein that we've started placing elsewhere? https://github.com/artsy/emission/blob/master/src/lib/Scenes/Home/Components/ForYou/Components/ArtworkCarousel.tsx#L243

Reply:
Oh actually, thats already handled in the method! Ignore